### PR TITLE
Test request handler additions

### PR DESCRIPTION
**Pull request by omkhegde**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cfn/cfn.go [hunk 1]
```diff
@@ -121,6 +121,12 @@ func makeEventFunc(h Handler) eventFunc {
 		}
 		request := handler.NewRequest(
```
Comment (brianterry) [inline cfn/cfn.go:122]: We need to create a more idiomatic to pass in this information.

File: cfn/cfn.go [hunk 2]
```diff
@@ -121,6 +121,12 @@ func makeEventFunc(h Handler) eventFunc {
...
 			event.RequestData.LogicalResourceID,
+			event.NextToken,
```
Comment (brianterry) [inline cfn/cfn.go:124]: This will not compile, because the event type needs to be updated.

File: cfn/handler/request.go [hunk 3]
```diff
@@ -35,9 +53,15 @@ type Request struct {
 }
 
 // NewRequest returns a new Request based on the provided parameters
-func NewRequest(id string, ctx map[string]interface{}, sess *session.Session, previousBody, body []byte) Request {
+func NewRequest(id string, NextToken string, StackId string, StackTags map[string]string, Region string, AccountID string, SystemTags map[string]string, ctx map[string]interface{}, sess *session.Session, previousBody, body []byte) Request {
```
Comment (brianterry) [inline cfn/handler/request.go:56]: Same thing here, we need a better way to pass in this information. I suggest creating a new type.

**Comment by omkhegde** [comment]
Brian already created PR with correct fix

**Pull request closed without merging**